### PR TITLE
feat: improve player feeling

### PR DESCRIPTION
**Pull request by Rinder5**
added run button
increased player move speed
added delay before jumping (hopefully this will make it feel more weighty)
inverted mouse camera control

also, while testing this, I noticed that when I jumped over a gravel block it still disappeared. Is this supposed to happen?

### CODE REVIEW

**Comment by nicholaschuayunzhi** [comment]
> also, while testing this, I noticed that when I jumped over a gravel block it still disappeared. Is this supposed to happen?

Definitely not intended. Seems like a raycast length problem.

**Comment by nicholaschuayunzhi** [comment]
@Rinder5 I'm looking to restructure player controller for optimization. Can we discuss soon?

File: Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -6,9 +6,12 @@ public class PlayerController : MonoBehaviour
     public LayerMask Layer;
 
     [SerializeField]
-    private const float MoveSpeed = 2f;
-    private const float JumpForce = 5.2f;
+    public const float MoveSpeed = 2.5f;
+    public const float JumpDelay = 0.2f;
+    public const float JumpForce = 5.2f;
+    public const float RunMultiplier = 1.3f;
```
Comment (nicholaschuayunzhi) [inline Assets/Scripts/PlayerController.cs]: I dont think these const floats can be viewed in the inspector

Comment (Rinder5) [reply Assets/Scripts/PlayerController.cs]: Oh. Oh well, I don't think it's too necessary to be able to edit them anyway.

**Review by nicholaschuayunzhi — APPROVED** [review]

**Pull request merged**